solr/test-framework/src/java/org/apache/solr/cloud/SolrCloudTestCase.java

```diff
@@ -140,29 +138,18 @@ public static void shutdownCluster() throws Exception {
   @BeforeClass
   public static void setPrsDefault() {
     Class<?> target = RandomizedTest.getContext().getTargetClass();
-    if (target != null && target.isAnnotationPresent(NoPrs.class)) return;
-    if (isPRS()) {
-      System.setProperty("solr.prs.default", "true");
+    boolean prsDefault;
+    if (target != null && target.isAnnotationPresent(NoPrs.class)) {
+      prsDefault = false;
+    } else {
+      prsDefault = random().nextBoolean();
```

Comment:
shouldn't this **read** a sys prop so we can choose it for the build?

Reply:
Done, and added another sysProp to save the user-provided value that we might be overwriting because of Class annotations